pom.xml

```diff
@@ -1754,6 +1754,23 @@ under the License.
 							</rules>
 						</configuration>
 					</execution>
+					<execution>
+						<id>forbid-reload4j</id>
+						<goals>
+							<goal>enforce</goal>
+						</goals>
+						<configuration>
+							<rules>
+								<bannedDependencies>
+									<excludes>
+										<exclude>ch.qos.reload4j:reload4j</exclude>
+										<exclude>slf4j-reload4j</exclude>
```

Comment:
missing groupId; to my knowledge this shouldn't work as-is.

Reply:
Thanks, fixed